packages/nimble-components/src/button/tests/button-matrix.stories.ts

```diff
@@ -81,3 +81,23 @@ export const hiddenButton: StoryFn = createStory(
 export const textCustomized: StoryFn = createMatrixThemeStory(
     textCustomizationWrapper(html`<${buttonTag}>Button</${buttonTag}>`)
 );
+
+// The baseline of the elements should be aligned when positioned inline.
+export const inlineAlignment: StoryFn = createStory(
+    html`<span style="text-decoration: underline;">Text</span>
+        <nimble-button
```

Comment:
we should use `buttonTag` rather than `nimble-button` to get automatic updates in the unlikely event of a name change and to get a proper import tree if we ever move this story somewhere else.

Reply:
Same for the icon element.